.github/workflows/buildcheck.yml

```diff
@@ -13,29 +13,18 @@ on:
       - 'lib.src/**'
       - 'src/**'
       - '.github/workflows/**/*.yml'
```

Comment:
Hier könnten wir noch `.github/actions/**/*.yml` hinzufügen, oder?

Reply:
Mein Vorschlag wäre diese ganze Liste zu entfernen und den buildcheck immer zu machen. Er kostet nichts und schadet auch nicht. Stattdessen würde ich die permissions auf content: read setzen und den checkout auch absichern. Soweit ich weiß, haben wir keine secrets projekt, die man raustragen könnte.

Reply:
Naja, es gibt schon eine ganze Menge Verzeichnisse im Root, die beim kompilieren überhaupt nicht benutzt werden. Warum dann darauf triggern?
Füge bitte die actions hinzu und lass uns die permissions in einem weiteren PR machen. Dann können wir diesen PR schon mal mergen.